src/assessments/seo/subheadingsKeywordAssessment.js

```diff
@@ -92,17 +114,49 @@ class SubHeadingsKeywordAssessment extends Assessment {
 	 * @returns {string} The translated string.
 	 */
 	translateScore( score, subHeadings, i18n ) {
-		if ( score === this._config.scores.multipleMatches || score === this._config.scores.oneMatch ) {
+		let minNumberOfSubheadings = subHeadings.count * this._config.lowerBoundary;
+		let maxNumberOfSubheadings = subHeadings.count * this._config.upperBoundary;
+
+		if ( subHeadings.matches === 0 ) {
```

Comment:
You're not using the scores you've set above. You're repeating all if-statements instead. Why?